README.md

```diff
@@ -86,7 +86,18 @@ From a peer-to-peer alternative to github and similar third party git portal ser
 
 # Development Setup
 ## Prerequisites
-As well as NodeJS (v12 or later) and Rust you need the Rust `wasm32-wasi` target:
+* [NodeJS](https://nodejs.org/en/) (v12 or later)
+* [Rust](https://www.rust-lang.org/tools/install)
+* [Rustup](https://rustup.rs/)
+
+### Compiling on Windows and other Platforms
+As suggested on the Rust install page, a compiler such as the one included with Windows Visual Studio may be required.
+For Windows, Visual Studio Build Tools 2019 requires a Windows SDK for your platform via [Visual Studio Installer](https://visualstudio.microsoft.com/downloads/). Choose Modify, C++ Build Tools, then add the Windows SDK.
+A console compiler such as [MingW](http://mingw-w64.org/doku.php/download)'s GCC (GNU Compiler Collection) supports most other platforms.
+Alternative IDEs for writing Rust such as [Atom](https://atom.io/) which have [Rust support](https://atom.io/packages/ide-rust) are available.
+
```

Comment:
I like this, it's clear and instructive. I suggest we keep the common sections together, so move the block about other OS below the "WebAssembly (Wasm)" section, just above "Get the code".

Reply:
All done.